core/arch/arm/include/mm/core_mmu.h

```diff
@@ -19,8 +19,8 @@
 
 /* A small page is the smallest unit of memory that can be mapped */
 #define SMALL_PAGE_SHIFT	12
-#define SMALL_PAGE_MASK		(SMALL_PAGE_SIZE - 1)
 #define SMALL_PAGE_SIZE		BIT(SMALL_PAGE_SHIFT)
+#define SMALL_PAGE_MASK		((paddr_t)(SMALL_PAGE_SIZE - 1))
```

Comment:
I think the inner parentheses are not necessary?
```
((paddr_t)(SMALL_PAGE_SIZE - 1))
          ^                   ^
           \-------------------\-- here
```

Reply:
You're right, I'll fix.